Automattic/wp-calypso
# Use the page template modal package

#### Changes proposed in this Pull Request

Split out from https://github.com/Automattic/wp-calypso/pull/49661
Now that the page template modal has been added, update the ETK use it

#### Testing instructions

upload this to your sandbox with `yarn dev --sync` in the ETK directory
open a sandboxed site and navigate to the new page window.
The new page selector should work as before
Depends on https://github.com/Automattic/wp-calypso/pull/49661
